specs/templates/custom-component.md

```diff
@@ -65,13 +65,19 @@
 
```

Comment:
@jattasNI reviving this to get it in if we think it's still valuable

Reply:
D'oh, I thought I tagged you in this comment when I first put up the PR but accidentally left it in a pending state so it never sent.

> @rajsite The contents of this PR are from my memory of our meeting about the button group plus a few minutes of web searching. I got the sense that you had more in depth requirements in mind. If so, feel free to build on this PR. If you don't have time to do that and you think this is a step in the right direction then we could also submit this and create an issue to track further improvements.

Based on your approval it appears you think it's a step in the right direction so I'll clean it up and complete it. Can you think of anything else we should track in a follow up?

Reply:
Think it's good for where we are today. Expecting we'll need to revisit once the spright vision is more ironed out / the what belongs in the nimble library vs spright library idea is clearer